tests/server/test_server.py

```diff
@@ -69,6 +74,17 @@ def execute_side_effect(self, query: str, params: list | tuple) -> None:
                     old_user = self._db_users[index]
                     # noinspection PyTypeChecker
                     self._db_users[index] = old_user[:3] + (old_user[3] - params[0],) + old_user[4:]
+            case QUERIES.WALLET_BUY_ADD_BTC:
+                try:
+                    index = [user[0] for user in self._db_users].index(params[1])
+                except ValueError:
+                    raise psycopg.IntegrityError()
+                else:
+                    old_user = self._db_users[index]
+                    self._db_users[index] = (
+                        old_user[:3] + (old_user[3] - params[0] * 20000,) + old_user[4:]
```

Comment:
Is multiplication by 20000 correct here after the changes?

Reply:
No